apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -0,0 +1,247 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { type CourseRequestRow } from './instructorRequestCourse';
+
+export const RequestCourse = ({
+  course_requests,
+  resLocals,
+}: {
+  course_requests: CourseRequestRow[];
```

Comment:
Please call this variable `rows`. We reserve the name `course_requests` for arrays of type `CourseRequest` and we use `rows` for the generic name on pages that show rows of things.